js/interface.js

```diff
@@ -973,6 +973,17 @@ $('[data-push-save]').on('click', function() {
   savePushData();
 });
 
+// Scroll accordion tab to the top
+$('.panel-collapse').on('shown.bs.collapse', function () {
+  var $panel = $(this).closest('.panel');
+
+  if (!$panel || !$panel.offset()) {
+    return;
+  }
+
+  Fliplet.Studio.emit('accordionScroll', $panel.offset().top);
```

Comment:
why `accordionScroll` if this is meant to scroll the whole body (as per your comment)?

I would have expected a generic `scrollTo` name.

Reply:
fixed